protocol-designer/src/components/StepEditForm/fields/LabwareLocationField/index.tsx

```diff
@@ -20,18 +23,32 @@ export function LabwareLocationField(
   const labwareEntities = useSelector(getLabwareEntities)
   const robotState = useSelector(getRobotStateAtActiveItem)
   const moduleEntities = useSelector(getModuleEntities)
+  const additionalEquipmentEntities = useSelector(
+    getAdditionalEquipmentEntities
+  )
+  const hasWasteChute = getHasWasteChute(additionalEquipmentEntities)
   const isLabwareOffDeck =
     props.labware != null
       ? robotState?.labware[props.labware]?.slot === 'offDeck'
       : false
+  const displayWasteChuteValue =
+    props.useGripper && hasWasteChute && !isLabwareOffDeck
 
   let unoccupiedLabwareLocationsOptions =
     useSelector(getUnocuppiedLabwareLocationOptions) ?? []
 
-  if (props.useGripper || isLabwareOffDeck) {
+  if (isLabwareOffDeck && hasWasteChute) {
+    unoccupiedLabwareLocationsOptions = unoccupiedLabwareLocationsOptions.filter(
+      option => option.value !== 'offDeck' && option.value !== WASTE_CHUTE_SLOT
+    )
+  } else if (props.useGripper || isLabwareOffDeck) {
     unoccupiedLabwareLocationsOptions = unoccupiedLabwareLocationsOptions.filter(
       option => option.value !== 'offDeck'
     )
+  } else if (!displayWasteChuteValue) {
+    unoccupiedLabwareLocationsOptions = unoccupiedLabwareLocationsOptions.filter(
+      option => option.name !== 'D3 - Waste Chute'
```

Comment:
The sandbox shows `Waste Chute in D3` in selectors whether I checked checkbox or not.

Tested the following code.
Or maybe using `includes('Waste Chute')

```suggestion
      option => option.name !== 'Waste Chute in D3'
```

![Screenshot 2023-09-20 at 5 08 24 PM](https://github.com/Opentrons/opentrons/assets/474225/4b11747a-a004-4173-9da6-c4d6179ae72e)
![Screenshot 2023-09-20 at 5 08 35 PM](https://github.com/Opentrons/opentrons/assets/474225/048eb141-ae4a-478c-baf5-e0543bc86cb2)

Reply:
oops thanks for noticing this! i forgot that i changed the string name